tests/safety/test_subaru.py

```diff
@@ -6,13 +6,39 @@
 from panda.tests.safety.common import CANPackerPanda
 
 
-class TestSubaruSafety(common.PandaSafetyTest, common.DriverTorqueSteeringSafetyTest):
-  TX_MSGS = [[0x122, 0], [0x221, 0], [0x321, 0], [0x322, 0], [0x323, 0]]
-  STANDSTILL_THRESHOLD = 0  # kph
-  RELAY_MALFUNCTION_ADDR = 0x122
-  RELAY_MALFUNCTION_BUS = 0
-  FWD_BLACKLISTED_ADDRS = {2: [0x122, 0x321, 0x322, 0x323]}
-  FWD_BUS_LOOKUP = {0: 2, 2: 0}
+MSG_SUBARU_Brake_Status     = 0x13c
+MSG_SUBARU_CruiseControl    = 0x240
+MSG_SUBARU_Throttle         = 0x40
+MSG_SUBARU_Steering_Torque  = 0x119
+MSG_SUBARU_Wheel_Speeds     = 0x13a
+MSG_SUBARU_ES_LKAS          = 0x122
+MSG_SUBARU_ES_Brake         = 0x220
+MSG_SUBARU_ES_Distance      = 0x221
+MSG_SUBARU_ES_Status        = 0x222
+MSG_SUBARU_ES_DashStatus    = 0x321
+MSG_SUBARU_ES_LKAS_State    = 0x322
+MSG_SUBARU_ES_Infotainment  = 0x323
```

Comment:
Now that I'm seeing this in Python, maybe it makes sense to use an Enum for this, but the `.value` will be cumbersome, so maybe just a class? And a struct/enum in C. What do you think?